test/lib/redis-store-spec.js

```diff
@@ -6,460 +6,466 @@ var assert = require('assert');
 var redisCache;
 var customRedisCache;
 
-before(function () {
-  redisCache = require('cache-manager').caching({
-    store: redisStore,
-    host: config.redis.host,
-    port: config.redis.port,
-    auth_pass: config.redis.auth_pass,
-    db: config.redis.db,
-    ttl: config.redis.ttl
-  });
-
-  customRedisCache = require('cache-manager').caching({
-    store: redisStore,
-    host: config.redis.host,
-    port: config.redis.port,
-    db: config.redis.db,
-    ttl: config.redis.ttl,
-    isCacheableValue: function (val) {
-      // allow undefined
-      if (val === undefined)
-        return true;
-      // disallow FooBarString
-      else if (val === 'FooBarString')
-        return false;
-      return redisCache.store.isCacheableValue(val);
-    }
-  });
-});
-
-describe ('initialization', function () {
+describe('Normal Tests', function () {
 
-  it('should create a store with password instead of auth_pass (auth_pass is deprecated for redis > 2.5)', function (done) {
-    var redisPwdCache = require('cache-manager').caching({
+  before(function () {
+    redisCache = require('cache-manager').caching({
       store: redisStore,
       host: config.redis.host,
       port: config.redis.port,
-      password: config.redis.auth_pass,
+      auth_pass: config.redis.auth_pass,
```

Comment:
`auth_pass` is being easily deprecated we should promote usage of `password`. https://github.com/NodeRedis/node_redis#options-object-properties